### ISSUE TITLE
Key hyperparameters to reproduce results？

### ISSUE THREAD

**Issue by VickiCui** [GEM-benchmark/GEM-metrics#96]
Thanks for your great work!
I'm having some trouble when reproducing the results. Although trained as in the tutorial, there is still a big gap between the model I trained and the results given in the paper. **Can you provide the necessary hyperparameters in order to reproduce the results?** Such as batch size, epoch, learning rate, beam size, and so on. Thx!!!

**Comment by tuetschek** [comment]
@VickiCui I don't think this is the correct place – the repo here is just for the metrics scripts, it doesn't include the baseline submissions. Perhaps @sebastianGehrmann or @yjernite would know more on the baselines?

**Comment by VickiCui** [comment]
您好，您的邮件我已经收到，我会第一时间处理。

**Comment by sebastianGehrmann** [comment]
Which task are we talking about? Most of the models were trained for https://openreview.net/forum?id=CSi1eu_2q96 and the hyperparameters from that paper should replicate results.
